src/main/kotlin/Commands.kt

```diff
@@ -97,26 +97,29 @@ val aliveInfoCommand = command("👥 Живые игроки", "liveInfo", 3)
 
 val updateCheckCommand = command("", "updateCheck", 2)
 
-val hostRequestCommand = command("Запросы на ведение", "hostRequests", 1)
-val hostSettingsCommand = command("Список ведущих", "hostSettings", 1)
-val adminSettingsCommand = command("Список администраторов", "adminSettings", 1)
-val gamesSettingsCommand = command("Список игр", "gamesSettings", 1)
-val hostAdminSettingsCommand = command("Список настроек ведущих", "hostAdminSettings", 1)
-val advertCommand = command("Реклама", "advert", 0)
+val hostRequestCommand = command("Запросы на ведение", "hostRequests", 2)
```

Comment:
Can't check it right now, but I have a feeling that I've seen a use of param with index 2 for some of this buttons, yet they say that they only take 2 params